dependencies.props

```diff
@@ -27,7 +27,7 @@
   <!-- Package dependency verification/auto-upgrade configuration. -->
   <PropertyGroup>
     <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
-    <DependencyBranch>master</DependencyBranch>
+    <DependencyBranch>feature/utf8string</DependencyBranch>
```

Comment:
Note this is also used for the BuildTools version on line 70 below:

`<BuildInfoPath>$(BaseDotNetBuildInfo)buildtools/$(DependencyBranch)</BuildInfoPath>`

Do you have this branch in buildtools as well?

Reply:
it's not in buildtools, I did this as well as manually running the update in https://github.com/dotnet/core-setup/pull/4398